README.md

```diff
@@ -397,99 +517,193 @@ Also if you like to automatically record the execution time of different cells
 [nbextensions](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/install.html).
 
 A sample notebook is provided at [`dwh/test_spark.ipynb`](dwh/test_spark.ipynb)
-which uses `matplotlib` too, so to run this you need to install it too:
-```
-(venv) $ pip3 install matplotlib
-```
-At the end of this notebook, we wrap up the tests into reusable functions that
-we can use outside the Jupyter environment e.g., for automated indicator
-computation.
+which uses `matplotlib` too, so to run this you need to install it too: `(venv)
```

Comment:
Is there a particular reason that you preferred the inline version of the command to the original ``` version?

Reply:
Not intended, I think it was an issue with the auto-formatter. I've changed it back.